phase2_start_service_processes() called -- currently a NOP
phase3_start_service_processes() called -- currently a NOP
phase4_start_service_processes() called -- currently a NOP
phase5_start_service_processes() called -- currently a NOP
testcase_main(): started
EXPECTATION: The simulation should be terminated as soon as fork() is called, since we are not in kernel mode.
ERROR: Someone attempted to call fork1 while in user mode!
TESTCASE ENDED: Call counts:   check_io() 0   clockHandler() 0
